src/ucp/core/ucp_mm.h

```diff
@@ -18,6 +18,11 @@
 #include <inttypes.h>
 
 
+/* Remote keys with that many remote MDs or less would be allocated from a
+ * memory pool.
+ */
+#define UCP_RKEY_MPOOL_MAX_MD     3
```

Comment:
why is it 3? For instance MAX_RNDV_LANES is 4

Reply:
this is max number, not count

Reply:
with 3 it fits into 128 bytes (actually 112 bytes)
I guess we can tune it later on (either change that or MAX_RNDV_LANES), depending on other structures. Worse case, rndv on 4 lanes will use malloc

Reply:
I see, but AFAIU we'd like to avoid malloc in RNDV flow. Then would be good to align this value with RNDV lanes. Otherwise why not 2?

Reply:
i've tried to optimize it for cache line usage: it takes 8+32+24*N bytes
1->64
2->88
3->112
so 2 and 3 consume same amount of cache lines, and 1 is too little..

Reply:
ok